openlibrary/plugins/openlibrary/lists.py

```diff
@@ -1,20 +1,33 @@
 """Lists implementaion.
 """
 import random
+import six
 import web
 
 from infogami.utils import delegate
 from infogami.utils.view import render_template, public
 from infogami.infobase import client, common
 
-import six
-
-from openlibrary.core import formats, cache
+from openlibrary.core import cache
+from openlibrary.core import formats
+from openlibrary.core.lending import add_availability
 import openlibrary.core.helpers as h
 from openlibrary.utils import dateutil
 from openlibrary.plugins.worksearch import subjects
 
 
+@public
+def get_book_availability(lst):
```

Comment:
Let's move this function to the bottom of the file; `lists_home` and `class lists` are more logical at the top.